code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -18,22 +18,91 @@
 	display_name = "a pack of Midoris"
 	path = /obj/item/storage/fancy/cigarettes/cigpack_midori
 
+/datum/gear/marijuanapack
+	display_name = "a pack of medical Marijuana"
+	path = /obj/item/storage/fancy/cigarettes/cigpack_med
+
+/datum/gear/cigar
+	display_name = "Havana Cigar"
+	path = /obj/item/clothing/mask/cigarette/cigar/havana
+	cost = 2
+
+/datum/gear/smokingpipe
+	display_name = "smoking pipe"
+	path = /obj/item/clothing/mask/cigarette/pipe
+	cost = 2
+
 /datum/gear/lighter
 	display_name = "a cheap lighter"
 	path = /obj/item/lighter
 
+/datum/gear/matches
+	display_name = "a box of matches"
+	path = /obj/item/storage/box/matches
+
+/datum/gear/zippo
+	display_name = "zippo lighter"
+	path = /obj/item/lighter/zippo
+	cost = 2
+
+/datum/gear/candlebox
+	display_name = "a box candles"
+	description = "For setting the mood or for occult rituals."
+	path = /obj/item/storage/fancy/candle_box/eternal
+
 /datum/gear/rock
 	display_name = "a pet rock"
 	path = /obj/item/toy/pet_rock
 
+/datum/gear/camera
+	display_name = "a camera"
+	path = /obj/item/camera
+
+/datum/gear/redfoxplushie
+	display_name = "a red fox plushie"
+	path = /obj/item/toy/plushie/red_fox
+
+/datum/gear/blackcatplushie
+	display_name = "a black cat plushie"
+	path = /obj/item/toy/plushie/black_cat
+
+/datum/gear/voxplushie
+	display_name = "a vox plushie"
+	path = /obj/item/toy/plushie/voxplushie
+
+/datum/gear/lizardplushie
+	display_name = "a lizard plushie"
+	path = /obj/item/toy/plushie/lizardplushie
+
+/datum/gear/deerplushie
+	display_name = "a deer plushie"
+	path = /obj/item/toy/plushie/deer
+
+/datum/gear/carpplushie
+	display_name = "a carp plushie"
+	path = /obj/item/toy/carpplushie
+
 /datum/gear/sechud
 	display_name = "a classic security HUD"
 	path = /obj/item/clothing/glasses/hud/security
 	allowed_roles = list("Head of Security", "Warden", "Security Officer", "Security Pod Pilot", "Internal Affairs Agent")
 
-/datum/gear/matches
-	display_name = "a box of matches"
-	path = /obj/item/storage/box/matches
+/datum/gear/paperbin
+	display_name = "a paper bin"
+	description = "For those who can't speak or like to write."
+	path = /obj/item/paper_bin
+
+/datum/gear/cryaonbox
+	display_name = "a box of crayons"
+	path = /obj/item/storage/fancy/crayons
+
+/datum/gear/toykatana
```

Comment:
Not a fan of roundstart loadout items that look like deadly weapons.

Reply:
I'd be more worried if the real deadly weapon was in any way common. But seeing a katana is so rare, I don't think there's much risk of mix-ups. Besides, we have roundstart loadout items that make your ID look like an emag, so if that's okay I don't think this will cause much trouble.

Reply:
Wearing an emag ID decal won't get sec to shoot you. Wearing a (fake) katana just might.